src/main/scala/mesosphere/marathon/api/akkahttp/v2/TasksController.scala

```diff
@@ -100,7 +100,8 @@ class TasksController(
         tryParseTaskIds(taskIds) match {
           case Left(rejection) => reject(rejection)
           case Right(instanceIdsToAppId) =>
-            if (!isAuthorized(instanceIdsToAppId.values)) {
+            val affectedApps = instanceIdsToAppId.values.flatMap(appId => groupManager.app(appId))(collection.breakOut)
```

Comment:
What is the type of `affectedApps`? I don't quite get what `latMap(appId => groupManager.app(appId))` is doing. Also, I'm unsure if `collection.breakOut` works if there is no type on `affectedApps`. I'm probably missing something.

Reply:
@alenkacz is probably best to answer that..

Reply:
well.. I expect her to give an answer to the breakOut... the type is `Seq[AppDefinition]`
as it is used in private def kill function on L204

Reply:
`collection.breakOut` is for when you want the result of some operation to turn in to some other type. 

Recommend:
```
val affectedApps: Seq[AppDefinition] = instanceIdsToAppId.values.flatMap(appId => groupManager.app(appId))(collection.breakOut)
```

Reply:
I know what `breakOut` does but without a type annotation for `affectedApps` this becomes very hard to read. Unfortunately this has been merged. `val affectedApps: Seq[AppDefinition]` would have been cleaner IMHO.